carmart-tx/pom.xml

```diff
@@ -111,6 +111,12 @@
             <artifactId>infinispan-core</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-commons</artifactId>
+            <version>7.1.1.Final</version>
```

Comment:
This version setting should just be temporary until we can either add infinispan-commons into the bom, or refactor the quickstart so that it doesn't need this jar.